addons/hr_expense/models/hr_expense.py

```diff
@@ -602,7 +602,7 @@ def action_move_create(self):
                     Command.create({
                         'name': expense.employee_id.name + ': ' + expense.name.split('\n')[0][:64],
                         'quantity': expense.quantity or 1,
-                        'price_unit': expense.total_amount,
+                        'price_unit': expense.unit_amount if expense.unit_amount != 0 else expense.total_amount,
```

Comment:
Thanks for the fix 🙂 
Did you check that previous versions were not affected?
Thanks!

Reply:
@IT-Ideas , No, it's not affected in the previous version. Thanks